jobs/config.json

```diff
@@ -1838,7 +1838,7 @@
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-scale"
```

Comment:
I'm not sure about taking ownership of release branch tests. Can we skip those for now?